index.html

```diff
@@ -3,16 +3,18 @@
 
 <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="css/joel-style.css">
```

Comment:
Hey @joelrottenberg - we don't want to add a link to your personal stylesheet here in the main index.html page - just the link to your index page, which you did on line 17 (great!). Please remove line 6, add and commit the change and push your branch back up. :)